updates/99-fas.update

```diff
@@ -11,7 +11,8 @@
 # ACI to expose FAS user attributes
 dn: cn=users,cn=accounts,$SUFFIX
 add:aci: (targetattr = "nsAccountLock || fasTimezone || fasLocale || fasIRCNick || fasGPGKeyId || fasCreationTime || fasStatusNote")(targetfilter = "(objectclass=fasuser)")(version 3.0;acl "Read FAS user attributes";allow (compare,read,search) userdn = "ldap:///all";)
-add:aci: (targetattr = "fasTimezone || fasLocale || fasIRCNick || fasGPGKeyId")(version 3.0;acl "Users can modify their own FAS attributes";allow (write) userdn = "ldap:///self";)
+remove:aci: (targetattr = "fasTimezone || fasLocale || fasIRCNick || fasGPGKeyId")(version 3.0;acl "Users can modify their own FAS attributes";allow (write) userdn = "ldap:///self";)
```

Comment:
@tiran 

Is there any difference at all between doing an `add:aci`/`remove:aci` versus a single `replace:aci`?

I was looking at some of the upstream update files and they seem to do a mix...and in one case they do a `replace:aci` and then an `add:aci` with the same value as the replacement, which just seems weird.

Reply:
I prefer remove/add over replace because it is more readable than replace. replace also replaces an existing value and does not add a new value.

Reply:
Gotcha, thanks. Upstream there's a sequence (in 60-trusts.update) that goes like this:

* add
* replace
* add (with the exact same value as the right side of the replace)

I'm just curious what the reasoning is. Does the second add actually do anything?